app/schema/user.py

```diff
@@ -13,7 +13,16 @@ class User(BaseModel):
     email: str
     activated: bool = True
     avatar_url: str
+    language: str = "ua"
 
     model_config = ConfigDict(
         from_attributes=True,
     )
+
+
+class LanguageUpdate(BaseModel):
+    language: str
+
+
+class UsernameUpdate(BaseModel):
+    username: str
```

Comment:
This can be one Schema where all fields are optional.

Reply:
its better if language will be of type `UserPreferredLanguage`, so frontend will have less chances to send us unknown string as his language value